app/backend/src/createusers.py

```diff
@@ -0,0 +1,109 @@
+import csv
+import sys
+from random import random
+from couchers.models import (
+    Base,
+    Conversation,
+    FriendRelationship,
+    FriendStatus,
+    GroupChat,
+    GroupChatRole,
+    GroupChatSubscription,
+    Message,
+    MessageType,
+    Reference,
+    ReferenceType,
+    User,
+)
+from couchers import config
+
+import random
+from couchers.crypto import hash_password
+from couchers.servicers.api import hostingstatus2sql
+from couchers.utils import Timestamp_from_datetime, create_coordinate
+from pb.api_pb2 import _HOSTINGSTATUS
+from datetime import date
+from couchers.db import get_user_by_field, session_scope
+import logging
+from faker import Faker
+
+fake = Faker()
+
+logger = logging.getLogger(__name__)
+
+cities = []
+countries = set()
+worldpopulation = 0
+
+// data from https://simplemaps.com/data/world-cities
+file_name = 'src/data/worldcities.csv'
+with open(file_name, "r") as file:
+    reader = csv.reader(file)
+    try:
+        for row in reader:
+            if row[0] != 'city' and row[9] != '':
+                print(row[0])
+                cities.append(row)
+                worldpopulation += int(float(row[9]))
+                countries.add(row[4])
+    except csv.Error as e:
+        sys.exit('file {}, line {}: {}'.format(file_name, reader.line_num, e))
+
+useramount = 1000000
+users = []
+hashedpassword = hash_password('MAGA2020')
```

Comment:
Please don't...

Reply:
If you meant the password: I will change that now.

Reply:
Thanks :)